packaging/nuget/NServiceBus.Persistence.Sql.MsBuild.targets

```diff
@@ -1,12 +1,20 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <ScriptDirectory>$(ProjectDir)$(IntermediateOutputPath)NServiceBus.Persistence.Sql\</ScriptDirectory>
```

Comment:
So it does not work now (after move of the persistence itself to VS 2017) but it did work before?